lib/instance-detail-helper.js

```diff
@@ -63,13 +63,12 @@ function parseBuildInfo(resp) {
     query_engine: resp.queryEngine ? resp.queryEngine : null,
     raw: resp // Save the raw output to later determine if genuine MongoDB
   };
-  // cover both cases of detecting enterprise module, see SERVER-18099
-  if (resp.gitVersion && resp.gitVersion.match(/enterprise/)) {
-    res.enterprise_module = true;
-  }
-  if (resp.modules && resp.modules.indexOf('enterprise') !== -1) {
+
+  // Cover both cases of detecting enterprise module, see SERVER-18099.
+  if (getMongoDBBuildInfo.isEnterprise(resp)) {
```

Comment:
i think you can just do this on line 62 above:
```js
enterprise_module: getMongoDBBuildInfo.isEnterprise(resp)
```

Reply:
ah ty lol thinking about other things to code when coding makes my code 🙃